src/main/resources/application.yml

```
@@ -279,4 +279,6 @@ app:
     timeout: ${TRX_LOCK_SECONDS_TIMEOUT:180}
   threads:
     schedule-max-number: ${THREADS_SCHEDULE_MAX_NUMBER:1}
+  synchronous:
```

let's use more specific name
```suggestion
  synchronousTransactions:
```